### DISCUSSION TITLE
Regarding x-y axes of Pole Figure plots in MTEX

### DISCUSSION THREAD

**Discussion by aniketchakrabarty** [mtex-toolbox/mtex#946] [Ask Anything]
Dear Sir,

With your help we were able to plot Pole Figures directly from a text file which contained the three Euler angles from a grain. 
We had done it with the help of the following command:
ori = loadOrientation_generic('84 grains.txt','CS',cs,'SS',ss,'ColumnNames', {'Euler1' 'Euler2' 'Euler3'})
For our work we had incorporated an orthorhombic sample symmetry ('222'). 
Our aim in this current work is to compare the <100>,<011> and <111> Pole Figures of FCC Aluminium (Aluminium alloy : 3D printed AlSi10Mg) obtained from the experimental observations with a few grains (200 to 300 grains which are a result of swapping of grains or tessellation) which we will be taking as input (as an input texture) to a CPFEM model using ABAQUS.
For this purpose we are trying to see whether or not the PFs obtained from swapped 200-300 grains match the experimentally measured ebsd results.
Having shared with you the background, the problem that we are facing now is that the axes of the Pole Figures (x and y) are getting interchanged, when we plot them from a text file by the procedure as stated above. 
I will explain the problem with a figure :
This is the figure for the PF for experimentally observed results for 145906 grains
![image](https://user-images.githubusercontent.com/73643457/114293887-9e45cf80-9ab7-11eb-92ac-3a2e99f9cbd8.png)
The one below is the one obtained for swapped 300 grains (Please take a note that the x and y axes are not in the same position as it is for the experimental counterpart)
![image](https://user-images.githubusercontent.com/73643457/114293926-08f70b00-9ab8-11eb-8e60-81d103b68c60.png)
Please help us to solve the issue.

**Comment by ralfHielscher** [comment]
The alignment of the x-Axis can be controlled by the commands `plotx2east`, `plotx2north`, etc.
Just execute the command at the beginning of you session.

Ralf.
